content/blog/cdf-intro-uber-crane/index.md

```diff
@@ -55,44 +56,48 @@ Last but not least, our SaaS product FABR Infra offers great convenience for dev
 
 ## Summary
 
-The original post from Uber is a case of an engineering-heavy large scale-up re-thinking and re-architecting infrastructure to better serve the entire engineering organisation. Ultimately it's about reducing the time, cost, and risk around making engineering changes.
+The original post from Uber is a case of an engineering-heavy large scale-up re-thinking and re-architecting infrastructure to better serve the entire engineering organisation. Ultimately it is about reducing the time, cost, and risk around making engineering changes.
 
 In our view:
 
 - **IaC** is the centre piece of the implementation, enabling automation.
 - **Abstraction** is the keyword given the mix of on-prem + multi-cloud. Because asking developers to understand and then utilise different infrastructure components with different characteristics is considered unrealistic and unattainable. Resulting in a small platform or infrastructure team being a bottleneck.
 
-By adopting OpenFABR CDF (our OSS project) and FABR Infrastructure (our SaaS product), we think there are immediate benefits:
+By adopting [OpenFABR CDF](https://github.com/openfabr/cdf) (our OSS project) and [FABR Infra](https://fabrhq.com) (our SaaS product), we think there are immediate benefits:
 
 - It becomes drastically easier to develop, test, and package infrastructure components in such a mixed infrastructure environment. CDF is designed to make this task easier for infrastructure package authors.
 - The standardised configuration format defined by CDF's modular approach not only makes config and change management easier, but also offers a free abstraction layer between organisation-wide shared infrastructure platforms and developers in the same organisation.
-- Developer self-serve wasn't stressed enough. With the adoption of FABR Infra, developer experience around infrastructure self-service is drastically improved. Developers can easily discover and use top-level infrastructure components and services made available to them.
+- With the adoption of FABR Infra, developer experience around infrastructure self-service is drastically improved. Developers can easily discover and use top-level infrastructure components and services made available to them.
 
-*If you have infrastructure challenges similar to Uber, we would love to chat.*
\ No newline at end of file
+*If you have infrastructure challenges similar to Uber's, we would love to chat.*
```

Comment:
Probably should include a calendly or hubspot call booking link here?

Reply:
I intentionally toned down the call to action to be less direct sales because. Maybe what we should have is a contacts us button in the template which will be more forgiving. 

The real question here is, _who is out audience for this type of article?_ should have discussed that upfront. Guessing you were thinking VP Eng, CTO etc.?

Reply:
Re target audience, not necessarily, could be somebody in leadership but the detailed writeup in the example section also suits perfects for a practitioner.

Reply:
If the blog template has a contact link or something then it saves us from adding those links, also more consistent as we wouldn't need to retro-update the links in posts if they are changed